pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -78,3 +78,23 @@ type RemoveDuplicatesArgs struct {
 	Namespaces        *api.Namespaces
 	ExcludeOwnerKinds []string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type LowNodeUtilizationArgs struct {
+	metav1.TypeMeta
+
+	ThresholdPriority                 *int32
+	ThresholdPriorityClassName        string
+	NodeResourceUtilizationThresholds *api.NodeResourceUtilizationThresholds
```

Comment:
instead of a nested field, I would have just expanded it at `LowNodeUtilizationArgs` level.

In other words,
```
type LowNodeUtilizationArgs struct {
	UseDeviationThresholds bool
	Thresholds             ResourceThresholds
	TargetThresholds       ResourceThresholds
	NumberOfNodes          int
}
```

```
type LowNodeUtilizationArgs struct {
	UseDeviationThresholds bool
	Thresholds             ResourceThresholds
	NumberOfNodes          int
}
```

Reply:
@JaneLiuL one of the reasons to introduce v1alpha2 configuration is to inline all previous strategy specific params into a single `XXXArgs` data type as @a7i suggests. So we get a flat type.

Also, this is a great opportunity to rename `Thresholds` and `TargetThresholds` into more corresponding names. E.g. `LowThreshold` and `HighThreshold` to get better correspondence with under/over utilized nodes. Open to discussion here.

Reply:
ok, now i get your point, i suppose param not pass  ResourceThresholds to be before when @a7i mention.
i will update as below
```
type LowNodeUtilizationArgs struct {
	metav1.TypeMeta

	UseDeviationThresholds bool
	Thresholds             ResourceThresholds
	TargetThresholds       ResourceThresholds
	NumberOfNodes          int
}

```